docs/cli.rst

```diff
@@ -111,6 +111,22 @@ Renku Command Line
 
 .. automodule:: renku.cli.image
 
+.. _cli-doctor:
+
+``renku doctor``
+----------------
+
+.. automodule:: renku.cli.doctor
+
+.. _cli-migrate:
+
+``renku migrate``
+-----------------
+
+.. automodule:: renku.cli.migrate
+
+.. _cli-githooks:
+
```

Comment:
Should we also add ls-files?

Reply:
It should be part of dataset documentation (`renku.cli.dataset.__doc__`)